### PR TITLE
Allows DTX to taste the Dankest Weed

### PR DESCRIPTION
**Pull request by MrMcMahan**
Allows DTX to taste the dankest WEED

]https://sc.mogicons.com/share/weed-63.jpg

### CODE REVIEW

**Comment by DesolateG** [comment]
Eh?

**Comment by Aurorablade** [comment]
Can we not meme or please

**Comment by Birdtalon** [comment]
DTX will be pleased.

**Comment by LightFire53** [comment]
And this is how you get banned from the git hub. Don't post meme pull requests just so the text shows up in OOC Chat.

**Comment by Aracino** [comment]
And he's using my characters last name. Why exactly?

File: code/game/magic/Uristrunes.dm [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 
 var/list/rune_cache = list()
+
+
+DTX_CAN_TASTE_DANKEST_WEED == 1
```
Comment (FalseIncarnate) [inline code/game/magic/Uristrunes.dm]: At least put some effort into memes.

**Comment by FlattestGuitar** [comment]
I chuckled, but it was the bad kind of a chuckle.

**Comment by Birdtalon** [comment]
Somehow the fact that they joined Github 2 hours ago doesn't give me much hope that they would contribute again anyway. @LightFire53

**Comment by AffectedArc07** [comment]
What. The. Fuck.

**Comment by jughu** [comment]
You all got @pranked

**Comment by FalseIncarnate** [comment]
@jughu please do not mention people outside of the ss13 community. We had a crazylemon guy get really mad at us because someone did that on accident before.

**Comment by Lady-Luck** [comment]
dude weed lmao

**Comment by Spacemanspark** [comment]
@falseincarnate He wasn't really mad, honestly.
But ya, it's pretty rude to do that intentionally.

**Pull request closed without merging**